### ISSUE TITLE
Separate tool from content

### ISSUE THREAD

**Issue by jelovirt** [dita-ot/dita-ot#1261]
Converted from [SourceForge issue 3519843](http://sourceforge.net/support/tracker.php?aid=3519843), submitted by @mkarg

Currently the DITA-OT expects to find all [i. e. content (.dita, .ditaval), customization (.xsl) and tool (DITA-IT itself)] inside of the DITA-OT's file structure. This does not match professional production needs. Typically on modern desktops, tools and content are separated. For example, DITA-OT (the tool and the default stylesheets) are installed on %ProgramFiles% where only administrators have write access since Windows Vista, while content (topics, maps, etc.) are found on %AppData%, which ist the current user's profile where he stores all his private work.

While it is just a pity but not a real problem that the documentation does not tell how simply it is to set up a private content directory (one will find out soon), it is a problem that customization has to be put into DITA-OT (i. e. %ProgramFiles%) so (a) an administrator is needed and (b) it is shared by all projects of all users on this PC. Certainly I do not want to find conflicting extensions making me trouble in my project just because some other user needed to install an extension. Also, the extension mostly is dependend of the project but not of the PC, so it has to be installed on all PCs editing this project (which not necessarily is all PCs in a company).

To sum up, it would be great if the manual would tell how to put customization in %AppData% so it could stay with the content instead of asking the administrator to modify the local DITA-OT installation. A tool is a tool. Material is material. The need to have different colours in a PDF is a material change, not a tool change.
